mnemosyne.env

```diff
@@ -0,0 +1,18 @@
+# This file is read from /etc/default/mnemosyne
+# This can be modified to change the default setup of the unattended installation
+
+HPFEEDS_HOST=hpfeeds3
+HPFEEDS_PORT=10000
+MONGODB_HOST=mongodb
+MONGODB_PORT=27017
+IDENT=mnemosyne
+OWNER=chn
```

Comment:
You expose the OWNER variable here, but not in other instances such as hpfeeds-{cif,logger}. I would favor either hard-coding or exposing for all

Reply:
I would opt for not exposing it at all. It's not something I think users should be modifying, unless they have a very good and specific reason for doing so.